pynetdicom/association.py

```diff
@@ -606,6 +606,9 @@ def run(self):
             self._dul_ready.wait()
 
         if self.is_acceptor:
+            if _config.STORE_RECV_CHUNKED_DATASET_PATH:
+                os.mkdir(_config.STORE_RECV_CHUNKED_DATASET_PATH / self.dul.ident)
```

Comment:
@scaramallion wdyt about this? sort of tightly coupled to the current concurrency model (see https://werkzeug.palletsprojects.com/en/1.0.x/local/ for a similar idea that also works with greenlets)

i also think if the idea is "the user only has access to the stored file for the course of the association" then it might be better to just create a `TemporaryDirectory` in the DUL and make the DIMSEMessage aware of it with a thread local or whatever